packages/webview_flutter/webview_flutter_android/test/android_webview_controller_test.dart

```diff
@@ -770,6 +770,30 @@ void main() {
   });
 
   group('AndroidWebViewWidget', () {
+    late List<MethodCall> methodChannelLog;
+
+    setUpAll(() {
+      SystemChannels.platform_views.setMockMethodCallHandler(
+        (MethodCall call) async {
+          methodChannelLog.add(call);
+          if (call.method == 'resize') {
```

Comment:
This is quite fragile, since it relies on internal implementation details of the framework. Couldn't we do a trivial wrapper around `PlatformViewsService` and allow injecting a trivial mock for that wrapper instead?

Reply:
Thats a good suggestion since we already do this with the `AndroidProxy`. done